### PR TITLE
Resolve comments in PR 1571

### PR DESCRIPTION
**Pull request by liuzhe-lz**

### CODE REVIEW

File: src/sdk/pynni/tests/test_compressor.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 from unittest import TestCase, main
-import nni.compression.tensorflow as tf_compressor
-import nni.compression.torch as torch_compressor
+import tensorflow as tf
 import torch
 import torch.nn.functional as F
-import tensorflow as tf 
+import nni.compression.tensorflow as tf_compressor
+import nni.compression.torch as torch_compressor
```
Comment (QuanluZhang) [inline src/sdk/pynni/tests/test_compressor.py]: this import order cannot pass pylint

Comment (liuzhe-lz) [reply src/sdk/pynni/tests/test_compressor.py]: It can.

File: src/sdk/pynni/nni/compression/tensorflow/builtin_pruners.py [hunk 2]
```diff
@@ -32,8 +32,8 @@ class AGP_Pruner(Pruner):
     """
     def __init__(self, config_list):
         """
-            Configure Args
-                initial_sparsity:
+            Configure Args:
```
Comment (xuehui1991) [inline src/sdk/pynni/nni/compression/tensorflow/builtin_pruners.py]: please follow the docstring format like other tuners?

Comment (liuzhe-lz) [reply src/sdk/pynni/nni/compression/tensorflow/builtin_pruners.py]: Updated.
Followed format used in `msg_dispatcher.handle_report_metric_data`

Comment (xuehui1991) [reply src/sdk/pynni/nni/compression/tensorflow/builtin_pruners.py]: I think this format has some differences with the format that we used(We used the format like xgboost).   I not sure about that. @leckie-chn

Comment (liuzhe-lz) [reply src/sdk/pynni/nni/compression/tensorflow/builtin_pruners.py]: This does not document the parameter list, instead it documents the fields of one parameter.

**Comment by ultmaster** [comment]
Would you add a link to compressor docs in table of contents? Currently, there's no way to see compressor other than searching for it.

**Pull request merged**